src/ui/common/src/components/workflows/workflowStatus.tsx

```diff
@@ -123,10 +123,29 @@ export const StatusChip: React.FC<Props> = ({ status }) => {
 
 export default StatusChip;
 
+type IndicatorProps = {
+  /**
+   * Execution status to render.
+   */
+  status: ExecutionStatus;
+  /**
+   * Size of the Indicator.
+   */
+  size: string;
+  /**
+   * False if use default colors. Otherwise, specify the color.
+   */
+  monochrome: string | false;
```

Comment:
Would you mind adding an example to our Storybook story with these new props?

Please do so in StatusIndicatorStory.stories.tsx. You should be able to follow the existing examples that are already there and add one for the monochrome prop.

Reply:
Added!
<img width="1436" alt="Screen Shot 2023-01-13 at 6 30 26 AM" src="https://user-images.githubusercontent.com/30596854/212344404-ecd47ee2-9421-4ea0-8327-634e569e7d92.png">
<img width="1436" alt="Screen Shot 2023-01-13 at 6 30 23 AM" src="https://user-images.githubusercontent.com/30596854/212344405-3015ddcc-e2c1-4e5d-b885-06659026e2f6.png">

Reply:
Awesome!